ocp4/profiles/coreos-ncp.profile

```diff
@@ -140,75 +133,75 @@ selections:
 
     ### Kernel Config
     ## Boot prompt
-    #- grub2_audit_argument
-    #- grub2_audit_backlog_limit_argument
-    #- grub2_slub_debug_argument
-    #- grub2_page_poison_argument
-    #- grub2_vsyscall_argument
-    #- grub2_vsyscall_argument.role=unscored
-    #- grub2_vsyscall_argument.severity=info
-    #- grub2_pti_argument
+    - grub2_audit_argument
+    - grub2_audit_backlog_limit_argument
+    - grub2_slub_debug_argument
+    - grub2_page_poison_argument
+    - grub2_vsyscall_argument
```

Comment:
I understand the hardening potential of this setting, but IIRC vsyscalls were used in RHEL-6 and per Ben Briard, this is still a use-case we need to keep in mind. I'm not sure if can, for the purpose of the certification say "too bad, can't use this". If we can, then fine.

Reply:
``vsyscall`` is already disabled in other government profiles, such as the DoD baselines for RHEL.

Having this rule in the ComplianceAsCode catalog allows for per-profile selection. For example the Government baselines may require this, and accept the implications, however more general-purpose baselines like PCI-DSS the rule may not need to be selected.

Reply:
Right, it's fine to disable it in this profile. It's expected.